cime_config/buildnml

```diff
@@ -222,6 +248,8 @@ def prep_input(case, casedocsdir, datestamp, nmlgen_input, prefix, input_type):
     input_files["co2hist"] = [os.path.join(dir_fix_am,"global_co2historicaldata_glob.txt"), "ln", "co2historicaldata_glob.txt"]
     input_files["co2mont"] = [os.path.join(dir_fix_am,"co2monthlycyc.txt"), "ln", "co2monthlycyc.txt"]
 
+    # NOTE: CIME.namelist.parse returns single values as one element list
+    ico2 = list_to_value(ico2)
```

Comment:
it is not always list. If I read the namelist file with CIME.namelist.parse it is list (basically with one item for ico2 and iaer) but if use the nmlgen produced through the XML file it is not list and I thing adding [0] will fail in this case. I am not sure why this two approach creates different data types for same namelist option. This could be result of inconsistencies in the CIME interface.